test/controllers/partials/BtaHomeControllerSpec.scala

```diff
@@ -97,4 +97,64 @@ class BtaHomeControllerSpec extends ControllerBaseSpec {
       }
     }
   }
+
+  "Calling the .claimEnrolment action" when {
+
+    "A user is logged in and enrolled to HMCE-VATDEC-ORG" should {
+
+      val goodEnrolments: Enrolments = Enrolments(
+        Set(
+          Enrolment(
+            "HMCE-VATDEC-ORG",
+            Seq(EnrolmentIdentifier("VRN", "123456789")),
```

Comment:
The Enrolment Identifier for the old VAT enrolment is `VATRegNo`. It doesn't matter for these tests as you are just mocking a success regardless but might as well be business-accurate.